gravitee-apim-console-webui/src/shared/components/gio-license/gio-license.service.ts

```diff
@@ -16,67 +16,96 @@
 import { HttpClient } from '@angular/common/http';
 import { Inject, Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
-import { map, shareReplay } from 'rxjs/operators';
+import { map, shareReplay, tap } from 'rxjs/operators';
+import { action } from '@storybook/addon-actions';
```

Comment:
You can't have that dependency here. It's for storybook and not for the APIM console. 
maybe a bad copy and paste ?

Reply:
I've just seen this used elsewhere - you can simply delete the action and parent pipe. This dialog returns no value to be interpreted this is not necessary.